databases/week2/Databases Folder/Small_Business.sql

```diff
@@ -0,0 +1,30 @@
+create database Small_Business;
+use Small_Business;
+CREATE TABLE `Staff` (
```

Comment:
its a good idea to mention the staff table after Office, as the staff table use the PK belongs to Office

Reply:
done